src/Symfony/Component/Console/Style/SymfonyStyle.php

```diff
@@ -204,7 +205,16 @@ public function caution($message)
      */
     public function table(array $headers, array $rows)
     {
-        $headers = array_map(function ($value) { return sprintf('<info>%s</>', $value); }, $headers);
+        array_walk_recursive($headers, function (&$value) {
```

Comment:
`recursive` required? Then should we add a test case which would trigger this recursive part?